### DISCUSSION TITLE
Does iperf3 support multi-threading?

### DISCUSSION THREAD

**Discussion by mohaa7** [esnet/iperf#1570] [Q&A]
[Here](https://fasterdata.es.net/performance-testing/network-troubleshooting-tools/iperf/multi-stream-iperf3/) it's given that the new beta version of iperf3 is multithreaded, but the [FAQ](https://software.es.net/iperf/faq.html) says it's not. May you please clarify it.

**Comment by swlars** [comment]
We do have a multi-threaded beta we're planning on merging after we've done a bit more testing. We haven't updated the FAQ, since it isn't merged yet, but that's a good reminder.
